<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupDlg</class>
 <widget class="QDialog" name="GroupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Group</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>&amp;Name:</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="typeLabel">
           <property name="text">
            <string>&amp;Type:</string>
           </property>
           <property name="buddy">
            <cstring>typeLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="typeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="applyPushButton">
           <property name="text">
            <string>A&amp;pply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetPushButton">
           <property name="text">
            <string>Re&amp;set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="attributesLabel">
         <property name="text">
          <string>&amp;Attributes:</string>
         </property>
         <property name="buddy">
          <cstring>attributeTableWidget</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>A&amp;dd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="attributeTableWidget"/>
     </item>
     <item>
      <widget class="QLabel" name="docLabel">
       <property name="text">
        <string>&amp;Doc:</string>
       </property>
       <property name="buddy">
        <cstring>docTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="docTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>typeLineEdit</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>attributeTableWidget</tabstop>
  <tabstop>docTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
